### PR TITLE
repackage .deb with xz compression

### LINKED ISSUE #3154: "ZST" Compression Not Supported in Debian dpkg
### Describe the bug

It looks like the compressed files in the ".deb" file have changed from the "xz" format to "zst" format. Unfortunately "zst" does not appear to be supported by Debian dpkg:

image

image

image

That means Betaflight Configurator can no longer be installed on Debian OS without some kind of work around, or just going back to using "xz" for compression.

### To Reproduce

Fails in Debian 10 and 11

### Expected behavior

dpkg installs new Betaflight Configurator package.

### Configurator version

10.9.0-RC2

### Flight controller configuration

_No response_

### Add any other context about the problem that you think might be relevant here

_No response_

### PR DESCRIPTION
**Pull request by nerdCopter**
* repackage ubuntu's .deb file as xz rather than zst
* ref: https://gist.github.com/goproslowyo/fe95c8e07c73593e2099687eeda78bc5
* fixes #3154 
* i'm not certain i like this approach, but it solves the problem -- i'll leave it to BF team to decide.
* adds about 4 minutes to Linux build :(
* PR marked to "Allow edits"

### CODE REVIEW

**Comment by nerdCopter** [comment]
not correct yet, i'll work on it as time permits
image

**Comment by nerdCopter** [comment]
oops, i was wrong 494MB is appropriate:
image


also, it works on `Debian 11`:
image

**Comment by howels** [comment]
@nerdCopter this occurs because Ubuntu has updated dpkg-deb to use zstd but Debian has not. Suggest that we could save time by simply running the build inside a Debian image instead of Ubuntu. This would then not require the slow recompression step.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
@howels trying to get `gulp-debian` working locally. But maybe this might be of interest:

https://github.com/betaflight/betaflight-configurator/pull/3155#issuecomment-1358718000

**Comment by McGiverGim** [comment]
> @nerdCopter this occurs because Ubuntu has updated dpkg-deb to use zstd but Debian has not. Suggest that we could save time by simply running the build inside a Debian image instead of Ubuntu. This would then not require the slow recompression step. Another alternative would be to alias `dpkg-deb` to `dpkg-deb -Zxz` in the shell used to build.

Github actions only have Ubuntu as image, at least that is what I see in the documentation.

File: .github/workflows/ci.yml
Comment (McGiverGim) [inline .github/workflows/ci.yml:96]: Simply add a comment here about why we are doing this. It will be easier for future developers when they see this code.

File: .github/workflows/ci.yml
Comment (McGiverGim) [inline .github/workflows/ci.yml:110]: I think `false` is the default value, so remove this line.

**Review by McGiverGim — CHANGES_REQUESTED** [review]
I like this approach. We have one problem, and your solution (if it works) is a workaround, in one simple step, that we can add/remove easily in a future.
If we can't find another better option, this is ok to me.

**Review by McGiverGim — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
